ads/xlift.md

```diff
@@ -0,0 +1,37 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Xlift
+
+## Example
+
+```html
+<amp-ad width="300" height="250"
+    type="xlift"
+    data-mediaid="mamaster">
+</amp-ad>
+```
+
+## Configuration
+
+For configuration details and to generate your tags, please contact https://www.x-lift.jp/#contact
+
+Supported parameters:
+
+- width
```

Comment:
`width` and `height` are required for any amp-ad. You don't need to include them in your doc. Also you may want to indicate that `data-mediaid` is required.